homeassistant/components/moon/sensor.py

```diff
@@ -13,7 +13,26 @@
 
 DEFAULT_NAME = "Moon"
 
-ICON = "mdi:brightness-3"
+STATE_FIRST_QUARTER = "first_quarter"
+STATE_FULL_MOON = "full_moon"
+STATE_LAST_QUARTER = "last_quarter"
+STATE_NEW_MOON = "new_moon"
+STATE_WANING_CRESCENT = "waning_crescent"
+STATE_WANING_GIBBOUS = "waning_gibbous"
+STATE_WAXING_GIBBOUS = "waxing_gibbous"
+STATE_WAXING_CRESCENT = "waxing_crescent"
+
+MOON_ICONS = {
+    STATE_FIRST_QUARTER: "mdi:moon-first-quarter",
+    STATE_FULL_MOON: "mdi:moon-full",
+    STATE_LAST_QUARTER: "mdi:moon-last-quarter",
+    STATE_NEW_MOON: "mdi:moon-new",
+    STATE_WANING_CRESCENT: "mdi:moon-waning-crescent",   
```

Comment:
I looked at the error and it seems like you have spaces at the end of the line which need to be removed.
See https://dev.azure.com/home-assistant/Home%20Assistant/_build/results?buildId=16325&view=logs&j=4e214fed-f310-53fa-0018-b38e7beb405b&t=4e7ef364-9046-5187-cd06-98a96bf53c68&l=21